tests/gtests/dnnl_test_common.hpp

```diff
@@ -437,7 +437,13 @@ static void fill_data(const memory::dim nelems, const memory &mem, data_t mean,
 
 inline void fill_data(memory::data_type dt, const memory &mem, float mean,
         float deviation, double sparsity = 1.) {
-    size_t nelems = mem.get_desc().get_size() / memory::data_type_size(dt);
+    const auto dt_size = memory::data_type_size(dt);
+    if (dt_size <= 0) {
+        assert(!"unexpected data type");
```

Comment:
Is it considered an error or dt can be undef?

Reply:
`undef` dt is considered an error, as it's not clear what is the correct behavior in that case.